src/swarm/client/connection/RequestConnection.d

```diff
@@ -364,6 +382,8 @@ public abstract class IRequestConnection :
         assert(conn_pool !is null, typeof(this).stringof ~ ".ctor: connection pool interface must be non-null");
         assert(params !is null, typeof(this).stringof ~ ".ctor: request params instance must be non-null");
 
+        this.id = instance_count++;
```

Comment:
`this.id = typeof(this).instance_count++` or similar? Do we require `this.` on the static members?

Reply:
Updated.